include/libdnf/rpm/package_query.hpp

```diff
@@ -569,6 +569,13 @@ class PackageQuery : public PackageSet {
     // @replaces libdnf/sack/query.hpp:method:addFilter(int keyname, int cmp_type, int match) - cmp_type = HY_PKG_LATEST_PER_ARCH
     void filter_latest_evr(int limit = 1);
 
+    /// Group packages by `name` and `arch`. Then within each group, keep packages that correspond with up to `limit` of (all but) earliest `evr`s in the group.
+    ///
+    /// @param limit            If `limit` > 0, keep `limit` number `evr`s in each group.
+    ///                         If `limit` < 0, keep all **but** `limit` last `evr`s in each group.
+    /// @since 5.0
+    void filter_earliest_evr(int limit = 1);
```

Comment:
Do we really need it. According to description it looks like that that result of `filter_latest_evr(int limit = 1) is equal to result of `filter_earliest_evr(int limit = -1)`. Did I overlooked something?